t/t2408-sdbus-recovery.t

```diff
@@ -77,7 +77,20 @@ test_expect_success NO_CHAIN_LINT 'background subscription fails with EAGAIN' '
 	test_must_fail wait $pid &&
 	grep "Errno 11" signals.err
 '
+test_expect_success NO_CHAIN_LINT 'initiate another subscription' '
```

Comment:
commit message "check and add"?  Or perhaps just "add a" .. and i guess technically "test" should be plural :-)